### PR TITLE
openapi-request-validator: readme renamed validator variable (part of…

### PR DESCRIPTION
**Pull request by AlexHolly**
… #512)

### CODE REVIEW

File: packages/openapi-request-validator/index.ts [hunk 1]
```diff
@@ -206,6 +206,13 @@ export default class OpenAPIRequestValidator
...
 
+  public validateRequest(request) {
+    return this.validate(request);
+  }
+
+  /**
+   * @deprecated use validateRequest instead.
+   */
   public validate(request) {
     const errors = [];
```
Comment (jsdevel) [inline packages/openapi-request-validator/index.ts]: can you add a console.warn here and move the function body to validateRequest?

Comment (jsdevel) [reply packages/openapi-request-validator/index.ts]: also need to update the tests to use validateRequest in all projects

**Comment by AlexHolly** [comment]
It fails because the node_modules are not up to date, it should run fine after merging this PR.

**Comment by jsdevel** [comment]
@AlexHolly i only merge passing builds

**Comment by AlexHolly** [comment]
Do the tests use the local package?

@jsdevel 
> also need to update the tests to use validateRequest in all projects

This would only work after pushing the new version to npm.

Am I missing something? Should this be solved in two PR?

**Comment by jsdevel** [comment]
> This would only work after pushing the new version to npm.

ahh, yes you're correct on that

**Comment by AlexHolly** [comment]
looks like all cases have already been fixed.

**Pull request closed without merging**